### PR TITLE
DROTH-1768 Split Tierikisteri TrafficSign importer by trafficSign groups

### PR DESCRIPTION
**Pull request by MAC-CGI**
DROTH-1768 Split Tierikisteri TrafficSign importer by trafficSign groups

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/TrafficSignTierekisteriImporter.scala [hunk 1]
```diff
@@ -124,8 +124,72 @@ class TrafficSignTierekisteriImporter extends PointAssetTierekisteriImporterOper
...
+}
+class TrafficSignInformationSignsTierekisteriImporter extends TrafficSignByGroupTierekisteriImporter {
+  override val trafficSignGroup : TrafficSignTypeGroup = TrafficSignTypeGroup.InformationSigns
+}
+
+class TrafficSignServiceSignsTierekisteriImporter extends TrafficSignByGroupTierekisteriImporter {
+  override val trafficSignGroup : TrafficSignTypeGroup = TrafficSignTypeGroup.ServiceSigns
+}
+
+//TODO remove this code after merge US 1707
```
Comment (marquesfm) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/TrafficSignTierekisteriImporter.scala]: This TODO can be forgotten by mistake after deploying this.
Let's put this info on the tasks related to DROTH-1707 and on this task too, please.

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala [hunk 2]
```diff
@@ -222,7 +260,7 @@ object TierekisteriDataImporter {
   val tierekisteriDataImporters = Map[String, TierekisteriImporterOperations](
     "litRoad" -> litRoadImporterOperations,
     "roadWidth" -> roadWidthImporterOperations,
-    "trafficSign" -> trafficSignTierekisteriImporter,
+    //"trafficSign" -> trafficSignTierekisteriImporter,
```
Comment (marquesfm) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala]: Commented code

Comment (MAC-CGI) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala]: could stay commented, The main process continue work, not possible to call

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/TrafficSignTierekisteriImporter.scala [hunk 3]
```diff
@@ -124,8 +124,72 @@ class TrafficSignTierekisteriImporter extends PointAssetTierekisteriImporterOper
...
+  protected def expireAssets(linkIds: Seq[Long], signTypes: Set[Int]): Unit = {
+    val trafficSignsIds = trafficSignService.expireAssetsByLinkId(linkIds, signTypes, Some("batch_process_trafficSigns"))
   }
 }
+
+trait TrafficSignByGroupTierekisteriImporter extends TrafficSignTierekisteriImporter {
+  val trafficSignGroup: TrafficSignTypeGroup
+
+  def trafficSignsInGroup(trafficSignGroup: TrafficSignTypeGroup) = TrafficSignType.apply(trafficSignGroup)
+  //TODO uncomment this line after merge  US 1707
```
[the remark this answers is not in the document]
Comment (MAC-CGI) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/TrafficSignTierekisteriImporter.scala]: Will not append, you will have error

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/TrafficSignTierekisteriImporter.scala (hunk 1 above)
Comment (MAC-CGI) [reply digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/importer/TrafficSignTierekisteriImporter.scala]: Will not append, you will have error

**Pull request merged**